archaius2-core/src/main/java/com/netflix/archaius/annotations/PropertyInfo.java

```diff
@@ -23,6 +23,6 @@
 @Documented
 @Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
 @Target({ElementType.FIELD, ElementType.METHOD})
-public @interface Value {
+public @interface PropertyInfo {
```

Comment:
Ya, I don't like PropertyInfo.  However, I envision adding other attributes to this annotation.  Alternatively we can make the decision that PropertyName will only be used for naming and as such cannot have other attributes.  Additional attributes like immutability can be specified using a different annotation such as @Immutable.